source/usage-examples/deleteMany.txt

```diff
@@ -7,13 +7,15 @@ Delete Multiple Documents
 You can delete multiple documents in a collection by using the
 ``DeleteMany()`` method. 
 
-The following example specifies a query filter to the ``DeleteMany()``
-method, which matches documents in the ``movies`` collection where the
-``runtime`` field is greater than *800* and deletes all the matching
-documents. 
+Example
+-------
 
 .. include:: /includes/usage-examples/run-example-tip.rst
 
+The following example matches documents in the ``movies`` collection
+where the ``runtime`` field is greater than 800, deleting all 
```

Comment:
Question: are we ok with referring to field names as the value they contain? Or should we just say:
"where the runtime is greater than 800 [minutes]"?

Reply:
Confirmed that we should include the unit of measure in which it is applicable.

Reply:
Ah ok, I wasn't so concerned about the unit of measure which is why I included it in square brackets. The mainpoint here is disambiguating "field" from "value".